src/Databases/DatabaseReplicated.h

```diff
@@ -39,6 +41,7 @@ class DatabaseReplicated : public DatabaseAtomic
 {
 public:
     static constexpr auto ALL_GROUPS_CLUSTER_PREFIX = "all_groups.";
+    static constexpr auto REPLICA_UNSYNCED_MARKER = "UNSYNCED";
```

Comment:
Super minor, but should it be `_ UNSYNCED` since we add it at the end of the string with the server UUID?